projects/rulenode-core-config/src/lib/components/action/gps-geo-action-config.component.ts

```diff
@@ -62,7 +69,7 @@ export class GpsGeoActionConfigComponent extends RuleNodeConfigurationComponent
   }
 
   protected validatorTriggers(): string[] {
-    return ['fetchPerimeterInfoFromMessageMetadata', 'perimeterType'];
+    return ['fetchPerimeterInfoFromMessageMetadata', 'perimeterType', 'presenceMonitoringStrategyOnEachMessage'];
```

Comment:
Why add a new trigger if it doesn't change the form's validation or state?

Reply:
I have removed 'presenceMonitoringStrategyOnEachMessage' from validatorTriggers